<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>multiSchedEditDialogDouble</class>
 <widget class="QDialog" name="multiSchedEditDialogDouble">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="autogenerat">
         <property name="title">
          <string>auto generate:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>start</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>stop</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>step</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_start">
              <property name="toolTip">
               <string>start value for automatic value generation</string>
              </property>
              <property name="statusTip">
               <string>start value for automatic value generation</string>
              </property>
              <property name="minimum">
               <double>-99.989999999999995</double>
              </property>
              <property name="singleStep">
               <double>0.250000000000000</double>
              </property>
              <property name="value">
               <double>0.250000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_stop">
              <property name="toolTip">
               <string>stop value for automatic value generation</string>
              </property>
              <property name="statusTip">
               <string>stop value for automatic value generation</string>
              </property>
              <property name="minimum">
               <double>-99.989999999999995</double>
              </property>
              <property name="singleStep">
               <double>0.250000000000000</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_step">
              <property name="toolTip">
               <string>step value for automatic value generation</string>
              </property>
              <property name="statusTip">
               <string>step value for automatic value generation</string>
              </property>
              <property name="minimum">
               <double>0.010000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.250000000000000</double>
              </property>
              <property name="value">
               <double>0.250000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_generate">
              <property name="toolTip">
               <string>generate values</string>
              </property>
              <property name="statusTip">
               <string>generate values</string>
              </property>
              <property name="text">
               <string>generate</string>
              </property>
              <property name="icon">
               <iconset resource="../myresources.qrc">
                <normaloff>:/icons/icons/edit-add.png</normaloff>:/icons/icons/edit-add.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_member">
         <property name="title">
          <string>select member:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QListView" name="listView_member">
            <property name="toolTip">
             <string>list of available members</string>
            </property>
            <property name="statusTip">
             <string>list of available members</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>filter</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_filter">
              <property name="toolTip">
               <string>filter the list of available values by name</string>
              </property>
              <property name="statusTip">
               <string>filter the list of available values by name</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>values:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_insert">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>add new value</string>
           </property>
           <property name="statusTip">
            <string>add new value</string>
           </property>
           <property name="text">
            <string>add</string>
           </property>
           <property name="icon">
            <iconset resource="../myresources.qrc">
             <normaloff>:/icons/icons/edit-add.png</normaloff>:/icons/icons/edit-add.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_delete">
           <property name="toolTip">
            <string>delete selected value</string>
           </property>
           <property name="statusTip">
            <string>delete selected value</string>
           </property>
           <property name="text">
            <string>delete selected</string>
           </property>
           <property name="icon">
            <iconset resource="../myresources.qrc">
             <normaloff>:/icons/icons/edit-delete-6.png</normaloff>:/icons/icons/edit-delete-6.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget_values">
         <property name="toolTip">
          <string>list of values</string>
         </property>
         <property name="statusTip">
          <string>list of values</string>
         </property>
         <property name="columnCount">
          <number>1</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Values</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../myresources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>multiSchedEditDialogDouble</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
